OutOfSchool/OutOfSchool.WebApi/Services/ProviderServices/IPrivateProviderService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using OutOfSchool.WebApi.Models.Providers;
+
+namespace OutOfSchool.WebApi.Services.ProviderServices;
+
+/// <summary>
+/// Defines interface for functionality for Private Provider.
+/// </summary>
+public interface IPrivateProviderService
```

Comment:
(It's more question than recommendation) I see interface with one method, that are named differently. Is this i-face supposed to be extended by more methods? If not, than elaborate naming (of i-face and related class). Same for `IPublic...`